.github/workflows/create_release.yaml

```diff
@@ -37,7 +39,7 @@ jobs:
       - name: Create infrastructure components
         env:
           REGISTRY: "quay.io/capk"
-          IMAGE_NAME: "capk-manager-amd64"
+          IMAGE_NAME: "capk-manager"
```

Comment:
I don't think we want to change this, otherwise our manifests will not point to the amd64 image by default

Reply:
I guess I'm not sure why we would force the k8s manifests to point to the amd64 image by default instead of the multi-arch image manifest and allow the client to pick the appropriate architecture. There really shouldn't even be separate images anymore, https://github.com/jbpratt/cluster-api-provider-kubevirt/pkgs/container/capk%2Fcapk-manager/38860232?tag=v0.1.2-test-3 just different digests for architectures which is mapped by the image manifest.

 Happy to revert this if desired.

Reply:
> I guess I'm not sure why we would force the k8s manifests to point to the amd64 image by default instead of the multi-arch image manifest and allow the client to pick the appropriate architecture. 

do we have a multi-arch image?!

Reply:
```
❯ podman manifest inspect ghcr.io/jbpratt/capk/capk-manager:v0.1.2-test-3
{
    "schemaVersion": 2,
    "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
    "manifests": [
        {
            "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
            "size": 739,
            "digest": "sha256:650db04dcade3443916e0c7f4ba6d587df9cb1be3bdd6365c324b04bec3be336",
            "platform": {
                "architecture": "amd64",
                "os": "linux"
            }
        },
        {
            "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
            "size": 739,
            "digest": "sha256:f3bae6c06a04a4c159437c12d5b677ceba0463165294d58b6e7f9994b20a9396",
            "platform": {
                "architecture": "arm",
                "os": "linux"
            }
        },
        {
            "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
            "size": 739,
            "digest": "sha256:1cfa1530ab14b5d56bf04ce373de3133d834993105a991f4790f7e1fb2add329",
            "platform": {
                "architecture": "arm64",
                "os": "linux"
            }
        }
    ]
}

```

sorry, I didn't want to so confidently assert that the `-{arm64,amd64}` images aren't needed, but I don't see a need for them :smile_cat:

Reply:
All the work was already within the makefile, just not used (maybe for a reason, I'm fighting packer to build these arm64 capk images)  https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt/blob/43916429f4417c3b18ded7052768e5fb4265c0a6/Makefile#L230-L237

Reply:
oh nice! yeah I just tried this out myself and it works great.